bootstrap/file/file.go

```diff
@@ -0,0 +1,64 @@
+package file
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/interfaces"
+)
+
+func LoadFile(path string, timeout time.Duration, provider interfaces.SecretProvider) ([]byte, error) {
```

Comment:
Now that we moved this into `file` package, then function name should just be `Load` to avoid has in Go Lang is called name stuttering. (i.e. file.LoadFile)
```suggestion
func Load(path string, timeout time.Duration, provider interfaces.SecretProvider) ([]byte, error) {
```